test/TestExtensions/DefaultClusterFixture.cs

```diff
@@ -8,8 +8,11 @@ public class DefaultClusterFixture : BaseTestClusterFixture
         protected override TestCluster CreateTestCluster()
         {
             var options = new TestClusterOptions();
+#if !NETSTANDARD_TODO
+            // waiting for OrleansProviders project to be ported over to vNext
             options.ClusterConfiguration.AddMemoryStorageProvider("Default");
             options.ClusterConfiguration.AddMemoryStorageProvider("MemoryStore");
+#endif
```

Comment:
@xiazen  included this change from PR #2322 to slightly reduce your merge-conflict-required fixes.